rules/tax-invoice-vs-eftpos-receipt/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Understand the difference between Tax Invoice and Eftpos Receipt - A comprehensive guide to tax compliance and expense management.
 type: rule
-title: "Do you understand the difference: Tax Invoice vs. Eftpos Receipt?"
+title: Expenses - Do you understand the difference: Tax Invoice vs. Eftpos Receipt?
```

Comment:
Hi @JimmyChenSSW 

Not a big fan of a colon in the title

- [ ] 1. Please change to "Do you understand the difference between Tax Invoice and Eftpos Receipt?"

Reply:
Done - Agree.